src/TranslationUi/Term/MetaboxAction.php

```diff
@@ -49,13 +59,17 @@ public function save(Request $request, PersistentAdminNotices $notices): bool
         $remoteSiteId = $this->relationshipContext->remoteSiteId();
         $remoteTermId = $this->relationshipContext->remoteTermId();
 
-        $focuskw = $values["site-{$remoteSiteId}"][MetaboxFields::FIELD_FOCUS_KEYPHRASE];
-        $title = $values["site-{$remoteSiteId}"][Metaboxfields::FIELD_TITLE];
-        $metadesc = $values["site-{$remoteSiteId}"][Metaboxfields::FIELD_META_DESCRIPTION];
-        $canonical = $values["site-{$remoteSiteId}"][MetaboxFields::FIELD_CANONICAL];
+        $focuskw = $values["site-{$remoteSiteId}"][MetaboxFields::FIELD_FOCUS_KEYPHRASE] ?? '';
+        $title = $values["site-{$remoteSiteId}"][Metaboxfields::FIELD_TITLE] ?? '';
+        $metadesc = $values["site-{$remoteSiteId}"][Metaboxfields::FIELD_META_DESCRIPTION] ?? '';
+        $canonical = $values["site-{$remoteSiteId}"][MetaboxFields::FIELD_CANONICAL] ?? '';
+
+        $option = (array)(get_blog_option($remoteSiteId, 'wpseo_taxonomy_meta', []) ?: []);
```

Comment:
Casting to an array isn't necessary in this case, if `get_blog_option` returns the default value it will be an array, if it's return a falsy value because of  `?:` it will be an array.